### PR TITLE
add: PM round eligibility

### PR DESCRIPTION
**Pull request by cristinabaldor**
# Pull Request

## Summary & Motivation

[//]: # "When merged, this pull request will..."

add a cte to calculate which employees have had a leave within six weeks of lockbox date, and a case statement to cover eligibility requirements for all rounds.

## Self-review

- [ ] Update **due date**, **assignee**, and **priority** on our
      [TEAMster Asana Project](https://app.asana.com/0/1205971774138578/1205971926225838)

  _If this is a same-day request, please flag that in Slack_

- [ ] <kbd>Format</kbd> has been run on all modified files

- [ ] Ensure you are using the `union_dataset_join_clause()` macro for queries that employ any
      models from these datasets:
  - deanslist
  - edplan
  - iready
  - pearson
  - powerschool
  - renlearn
  - titan

## Troubleshooting

- [SqlFluff Rules Reference](https://docs.sqlfluff.com/en/stable/rules.html)
- [Trunk](https://teamschools.github.io/teamster/CONTRIBUTING/#trunk)
- [dbt](https://teamschools.github.io/teamster/CONTRIBUTING/#dbt-cloud_1)

### CODE REVIEW

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__schoolmint_grow_observation_details.sql [hunk 1]
```diff
@@ -10,6 +10,18 @@ with
         group by employee_number
     ),
 
+    recent_leave as (
+        select distinct
+            srh.employee_number, t.academic_year, t.code, true as recent_leave,
+        from {{ ref("base_people__staff_roster_history") }} as srh
+        inner join
+            `teamster-332318`.`kipptaf_reporting`.`stg_reporting__terms` as t
```
Comment (kevinverhoff) [inline src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__schoolmint_grow_observation_details.sql:18]: should this be the {{ ref()}} notation?

**Review by kevinverhoff — CHANGES_REQUESTED** [review]
this is really smart. I'm going to modify line 18 though

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__schoolmint_grow_observation_details.sql (hunk 1 above)
Comment (cristinabaldor) [reply src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__schoolmint_grow_observation_details.sql]: yep, my bad.

Comment (kevinverhoff) [reply src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__schoolmint_grow_observation_details.sql]: FIxed!

**Review by kevinverhoff — APPROVED** [review]
high fives all around

**Pull request merged**